crates/turbo-tasks-build/src/lib.rs

```diff
@@ -160,8 +160,10 @@ pub fn generate_register() {
 
 pub fn rerun_if_glob(globs: &str, root: &str) {
     let cwd = env::current_dir().unwrap();
-    let globs = cwd.join(globs);
-    let mut seen = HashSet::from([cwd.join(root)]);
+    let globs = cwd.join(globs.replace('/', PATH_SEP.to_string().as_str()));
+    let root = cwd.join(root.replace('/', PATH_SEP.to_string().as_str()));
+    println!("cargo:rerun-if-changed={}", root.display());
```

Comment:
Shouldn't that be handled by the `ancestors` iter above? Or is that for the case where `root` is empty?

Reply:
The root is in `seen` from the beginning, so it's ignored below